iron-lazy-pages-behavior.html

```diff
@@ -93,7 +93,7 @@
       if (this.selectedClass) {
         event.detail.item._toggleClass(this.selectedClass, true);
       }
-    }.bind(this), this);
```

Comment:
This is still required for https://github.com/TimvdLippe/iron-lazy-pages/pull/33/files#diff-06d57999e47ac40b3528de00729543c9R45